osu.Game/Overlays/Comments/CommentMarkdownContainer.cs

```diff
@@ -18,8 +18,10 @@ public class CommentMarkdownContainer : OsuMarkdownContainer
 
         private class CommentMarkdownTextFlowContainer : OsuMarkdownTextFlowContainer
         {
-            // Don't render image in comment for now
-            protected override void AddImage(LinkInline linkInline) { }
+            protected override void AddImage(LinkInline linkInline)
+            {
+                AddDrawable(new OsuMarkdownImage(linkInline));
+            }
         }
```

Comment:
I'm pretty sure all you need to do is remove this class and the place it's made completely. What you've done here is reimplemented this method in exactly the same way that it's implemented in the base class.

Reply:
I've removed the `CommentMarkdownTextFlowContainer` class and rather use the base class `OsuMarkdownTextFlowContainer`.

Reply:
Please don't mark things resolved, leave that to us.